firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -338,7 +338,10 @@ public boolean didCrashOnPreviousExecution() {
    * disabled. Use {@link #deleteUnsentReports()} to delete any reports stored on the device without
    * sending them to Crashlytics.
    *
-   * @param enabled whether to enable automatic data collection.
+   * @param enabled whether to enable automatic data collection. When set to `false`, the new value
```

Comment:
Suggest: 

`* @param specifies whether to enable automatic data collection...`